news/8187.feature.rst

```diff
@@ -0,0 +1,5 @@
+Run the wheel install in a multiprocessing Pool, this has x1.5 speedup factor
```

Comment:
Is there a concrete benchmark on this? I’m not sure we want to be this specific about the performance gain (to avoid false expectations). Something like “significant performance gain” would be enough for a changelog entry.

Reply:
Agreed